pkg/metrics/metrics.go

```diff
@@ -168,6 +168,34 @@ const (
 
 	// LabelDirection is the label for traffic direction
 	LabelDirection = "direction"
+
+	// LabelSourceCluster is the label for source cluster name
+	LabelSourceCluster = "source_cluster"
+
+	// LabelSourceNodeName is the label for source node name
+	LabelSourceNodeName = "source_node_name"
+
+	// LabelTargetCluster is the label for target cluster name
+	LabelTargetCluster = "target_cluster"
+
+	// LabelTargetNodeIP is the label for target node IP
+	LabelTargetNodeIP = "target_node_ip"
+
+	// LabelTargetNodeName is the label for target node name
+	LabelTargetNodeName = "target_node_name"
+
+	// LabelTargetNodeType is the label for target node type (local vs remote)
+	LabelTargetNodeType = "target_node_type"
+
+	// LabelType is the label for type in general
+	LabelType = "type"
+
+	LabelValueEndpoint = "endpoint"
+	LabelValueHTTP     = "http"
+	LabelValueICMP     = "icmp"
+	LabelValueLocal    = "local"
+	LabelValueNode     = "node"
+	LabelValueRemote   = "remote"
```

Comment:
These all have the prefix `LabelValue...`, however only some of them are alternatives for each other. I wonder if it would be easier to read & ensure that users correctly pick the right values by grouping alternatives with the same prefix, for example:

```suggestion
	LabelPeerEndpoint = "endpoint"
	LabelPeerNode     = "node"
	
	LabelTrafficHTTP     = "http"
	LabelTrafficICMP     = "icmp"
	
	LabelLocationLocal    = "local"
	LabelLocationRemote   = "remote"
```